framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
Maybe we could make the bindable properties of a command configurable? They are defined in https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/ui/core/MExecutable.js#L83-L92